### PR TITLE
[preview] add an option to prefer the preview to the editor.

### LINKED ISSUE #1456: [preview] Add option to prefer the preview over the editor
One may prefer to open the preview instead of the editor via double-click in the navigator, e.g. for markdown files. A double-click into the rendered preview already opens the editor. 

cc. @svenefftinge

### PR DESCRIPTION
**Pull request by AlexTugarev**
Adds "preview.preferToEditor" option.

Closes #1456

### CODE REVIEW

File: packages/preview/src/browser/preview-preferences.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  PreferenceContribution,
+  PreferenceSchema
+} from "@theia/core/lib/browser";
+
+export const PreviewConfigSchema: PreferenceSchema = {
+  type: "object",
+  properties: {
+    "preview.preferToEditor": {
+      type: "boolean",
+      description: "Open the preview instead of the editor by default.",
```
Comment (svenefftinge) [inline packages/preview/src/browser/preview-preferences.ts]: the description and the property name are inverse

Comment (svenefftinge) [reply packages/preview/src/browser/preview-preferences.ts]: I think it should be `preview.openByDefault` with a default value `false`.

Comment (AlexTugarev) [reply packages/preview/src/browser/preview-preferences.ts]: 'fine with renaming.

about the default value, I though we want to open the preview preferably to the editor in general. but I'm also fine with making it this way. just asking, I got it the other way around.

Comment (svenefftinge) [reply packages/preview/src/browser/preview-preferences.ts]: Ah yes. Let's try opening the preview by default.

Comment (AlexTugarev) [reply packages/preview/src/browser/preview-preferences.ts]: renaming done.

**Pull request merged**